src/pages/developers/tutorials/sui.mdx

```diff
@@ -209,92 +210,106 @@ sui client faucet
 This sends a faucet request to the local validator and credits your active
 address with test SUI.
 
-## Deposit from a Sui Contract
+## Deposit And Call from a Sui Contract
 
-In this step, you’ll call the `deposit` function from a deployed Sui contract,
-sending SUI tokens to a universal contract on ZetaChain.
+In Sui, calling a universal contract from a Move contract is done through a
+Programmable Transaction Block (PTB).
 
-### Get a Coin Object
+PTBs let you compose multiple actions into a single atomic transaction: you can
+call a Move function, capture its return value, and feed that value into the
+next call (such as Sui Gateway’s `deposit_and_call`). A client app (for example,
+a web app) builds and signs the PTB, submits it to Sui, and Sui executes the
+sequence as one transaction.
 
-First, find one of the SUI coin objects owned by your active address:
+In the created project, there’s a TypeScript command that builds such a PTB:
+`commands/suiDepositAndCall.ts`. By default, it ABI‑encodes a payload and sends
+it directly to the Sui Gateway. We’ll modify it to:
 
-```
-COIN=$(sui client objects \
-  $(sui client active-address) \
-  --json | jq -r '.[] | select(.data.type == "0x2::coin::Coin<0x2::sui::SUI>") | .data.objectId' | head -n 1) && echo $COIN
-```
+1. Call our Sui contract first and get its return value.
+2. Pass that returned value as the payload to the Sui Gateway.
 
-This sets `$COIN` to a valid token object you can spend.
+Open `commands/suiDepositAndCall.ts` and replace the payload construction:
 
-### Call the `deposit` Function
+```ts title="commands/suiDepositAndCall.ts"
```

Comment:
There's something off with this formatting.

<img width="693" height="174" alt="image" src="https://github.com/user-attachments/assets/31d77227-2f26-41f2-98e4-034b7cb25c77" />